api/web3server_utils.go

```diff
@@ -239,7 +244,7 @@ func (svr *Web3Server) getTransactionFromActionInfo(actInfo *iotexapi.ActionInfo
 			if err != nil {
 				return transactionObject{}, err
 			}
-			addr, err := ioAddrToEthAddr(receipt.ContractAddress)
+			addr, err := getExecutionContractAddr(receipt.ContractAddress)
```

Comment:
in `ioAddrToEthAddr`, for `input = ""`
```
return "0x0000000000000000000000000000000000000000", nil
``` 
now in `getExecutionContractAddr`, for `input = ""`
```
return "", nil
```
would this be an issue (behavior change for client)?

Reply:
Good finding. But I think there will never be a scenario where both  `to` and `contractAddr` are empty string